CMakeLists.txt

```diff
@@ -105,13 +105,18 @@ IF ( ENABLE_USE_THREADS )
   ENDIF ( CMAKE_USE_PTHREADS_INIT )
 ENDIF ( ENABLE_USE_THREADS )
 
+INCLUDE(CheckSymbolExists)
 FIND_PACKAGE(Rpm REQUIRED)
 IF ( NOT RPM_FOUND)
   MESSAGE( FATAL_ERROR " rpm-devel not found" )
 ELSE ( NOT RPM_FOUND)
   INCLUDE_DIRECTORIES(${RPM_INCLUDE_DIR})
   # fix includes not relative to rpm
   INCLUDE_DIRECTORIES(${RPM_INCLUDE_DIR}/rpm)
+  CHECK_SYMBOL_EXISTS(VERIFY_DIGEST rpmcli.h VERIFYDIGEST_FOUND)
```

Comment:
Looks like the test does not work. It fails to find VERIFY_DIGEST in rpmvf.h (same with CHECK_CXX_SYMBOL_EXISTS). 

It compiles on the CI system because it has rpm-4.14.2 installed. On SLE systems with rpm-4.14.1 it fails (no ::rpmtsSetVfyFlags).